src/OpenTelemetry.Instrumentation.Runtime/description.md

```diff
@@ -0,0 +1,198 @@
+| process.runtime.dotnet.**threadpool.completed.items.count** | ThreadPool Completed Work Item Count | `{items}`   | ObservableCounter | `Int64`    |                  |                  |
+| process.runtime.dotnet.**threadpool.queue.length**          | ThreadPool Queue Length              | `{items}`   | ObservableGauge   | `Int64`    |                  |                  |
+| process.runtime.dotnet.**active.timer.count**               | Number of Active Timers              | `{timers}`  | ObservableGauge   | `Int64`    |                  |                  |
+
+- [Monitor.LockContentionCount](https://docs.microsoft.com/dotnet/api/system.threading.monitor.lockcontentioncount?view=netcore-3.1):
+  Gets the number of times there was contention when trying to take the monitor's
+  lock.
+- [ThreadPool.ThreadCount](https://docs.microsoft.com/dotnet/api/system.threading.threadpool.threadcount?view=netcore-3.1):
+  Gets the number of thread pool threads that currently exist.
+- [ThreadPool.CompletedWorkItemCount](https://docs.microsoft.com/dotnet/api/system.threading.threadpool.completedworkitemcount?view=netcore-3.1):
+  Gets the number of work items that have been processed so far.
+- [ThreadPool.PendingWorkItemCount](https://docs.microsoft.com/dotnet/api/system.threading.threadpool.pendingworkitemcount?view=netcore-3.1):
+  Gets the number of work items that are currently queued to be processed.
+- [Timer.ActiveCount](https://docs.microsoft.com/dotnet/api/system.threading.timer.activecount?view=netcore-3.1):
+  Gets the number of timers that are currently active. An active timer is registered
+  to tick at some point in the future, and has not yet been canceled.
+
+## Process related metrics
+
+The metrics in this section can be enabled by setting the
+`RuntimeMetricsOptions.IsProcessEnabled` switch.
+
+| Name                    | Description                            | Units          | Instrument Type   | Value Type | Attribute Key(s) | Attribute Values |
+|-------------------------|----------------------------------------|----------------|-------------------|------------|------------------|------------------|
+| process.cpu.utilization | CPU utilization of this process        | `1`            | ObservableGauge   | `Double`   |                  |                  |
+| process.cpu.time        | Processor time of this process         | `s`            | ObservableCounter | `Int64`    | state            | user, system     |
+| process.cpu.count       | The number of available logical CPUs   | `{processors}` | ObservableGauge   | `Int64`    |                  |                  |
+| process.memory.usage    | The amount of physical memory in use   | `By`           | ObservableGauge   | `Int64`    |                  |                  |
+| process.memory.virtual  | The amount of committed virtual memory | `By`           | ObservableGauge   | `Int64`    |                  |                  |
```

Comment:
Note: `VirtualMemorySize64` is giving me numbers in the range of 2TB (numbers like 2204055367680, 2204056940544), when `WorkingSet64` is only 20~40 MB. I need to figure out how/why the `VirtualMemorySize64` is so big. (Probably something to do with how my runner is written.)

![image](https://user-images.githubusercontent.com/9310587/173145361-8bd1e4b8-63c2-4a1c-8ce9-e5106558f210.png)

Reply:
https://docs.microsoft.com/en-us/windows/win32/memory/working-set

Reply:
`VirtualMemorySize64 ` tells the total virtual memory available for the process to use (not necessarily be committed). While Working Set is `the set of pages in the virtual address space of the process that are currently resident in physical memory` from the doc that @reyang sent. 
These are two different things, and it is expected the Working Set always be less than the virtual memory size.